Jenkinsfile

```diff
@@ -17,7 +17,7 @@ pipeline {
       }
     }
     stage('Test') {
-      options { timeout(time: 10, unit: 'MINUTES') }
+      options { timeout(time: 15, unit: 'MINUTES') }
```

Comment:
If the average run time is 10+ mins, you can bump this to 20 mins to give us some margin

Reply:
done